lib/vrl/stdlib/src/parse_aws_vpc_flow_log.rs

```diff
@@ -209,80 +209,58 @@ fn parse_log(input: &str, format: Option<&str>) -> ParseResult<Value> {
 
 #[cfg(test)]
 mod tests {
-    // use super::*;
+    use super::*;
 
-    // vrl::test_type_def![
-    //     value_noop {
-    //         expr: |_| ParseAwsVpcFlowLogFn::new(Box::new(Noop), None),
-    //         def: TypeDef { fallible: true, kind: Kind::Map, ..Default::default() },
-    //     }
+    #[test]
+    fn parse_aws_vpc_flow_log() {
```

Comment:
Any reason we can't use the `test_function` macro here? Would probably be good for consistency to start using that wherever we can.